db/migrate/20160714020213_create_news_distributions.rb

```diff
@@ -0,0 +1,8 @@
+class CreateNewsDistributions < ActiveRecord::Migration[5.0]
+  def change
+    create_table :news_distributions do |t|
+      t.references :distribution, polymorphic: true, index: { name: 'index_news_distro_on_type_and_id' }
```

Comment:
It's not clear whether tags are restricted to Sections, or whether they are potentially anything. Polymorphic was chosen because it's more flexible and gives us the ability to support some of the tags that the prototypes have that aren't yet sections.